### DISCUSSION TITLE
node red example flows

### DISCUSSION THREAD

**Discussion by thewaver007** [vinodsr/node-red-contrib-tuya-smart-device#93] [General]
Hi, I am trying to use node red to turn on/off a gosund SP116 smart switch (same as a Tuya switch) and have tried to use your example nodered flow to no avail. I have the device IDs etc and can control it from the tuya iot website. Did you ever do an example nodered flow for this device as I am pulling my hair out trying to get it to work. I am a beginner but understand the basics of nodered but not very good with json. Kind regards Don...

**Comment by vinodsr** [comment]
Refer this for getting an idea about what all inputs options your device will support. 
https://github.com/vinodsr/node-red-contrib-tuya-smart-device/issues/87#issuecomment-1222199691

IMV There is no common rule for all tuya devices
